helmfile/charts/node-local-dns/templates/node-local-dns.yaml

```diff
@@ -115,9 +119,9 @@ spec:
       k8s-app: node-local-dns
   template:
     metadata:
-       labels:
-          k8s-app: node-local-dns
-       annotations:
+      labels:
+        k8s-app: node-local-dns
+      annotations:
         prometheus.io/port: "9253"
         prometheus.io/scrape: "true"
```

Comment:
I'm rather sure that these do not do anything, so could be removed. Any metrics from this should be gathered using a servicemonitor. But please double check that.
```suggestion
```

Reply:
How should I check that?